services/motion/motion.go

```diff
@@ -186,8 +186,117 @@ type PlanWithStatus struct {
 }
 
 // A Service controls the flow of moving components.
+//
+// Move example:
+//
+//	// Assumes a gripper configured with name "my_gripper" on the machine
+//	gripperName := gripper.Named("my_gripper")
+//
+//	geometriesInFrame := []*referenceframe.GeometriesInFrame{}
+//
+//	worldState, _ := referenceframe.NewWorldState(geometriesInFrame, nil)
+//
```

Comment:
If you aren't making a worldState you can delete these and just pass `nil` into the Move function.  If you want to include a worldstate then you should add geometries to it.  I'm not sure if the motivation of these docs is to include a minimal example or an example utilizing all the fields